Bottom: 7fe6c5cd33801f386bd13fe0148949f3c3fa44ac
Top:    df870f5bd477499bf8614e8308b36bc71dd18ab9
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-13 22:08:51 +0000

Refresh of attempt-to-fix-bad-rtree-usage

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 2392ff6..06cb7bc 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1969,8 +1969,8 @@ print_labels (PLINE * a)
   do
     {
       DEBUGP ("(%mn, %mn)->(%mn, %mn) labeled %s  is_round=%i radius=%f cvc_next=%p cvc_prev=%p\n",
-              EDGE_BACKWARD_VERTEX (e)->point[0], EDGE_BACKWARD_VERTEX (e)->point[1],
-               EDGE_FORWARD_VERTEX (e)->point[0],  EDGE_FORWARD_VERTEX (e)->point[1], theState (e));
+	      EDGE_FORWARD_VERTEX (e)->point[0],  EDGE_FORWARD_VERTEX (e)->point[1], theState (e),
+              e->is_round, e->radius, EDGE_BACKWARD_VERTEX (e)->cvc_next, EDGE_BACKWARD_VERTEX (e)->cvc_prev);
     }
   while ((e = NEXT_EDGE (e)) != &a->head);
 }
